### DISCUSSION TITLE
Save a directory list to a file

### DISCUSSION THREAD

**Discussion by bowdendata** [iterate-ch/cyberduck#15889] [Ideas]
I looking for a way in the CyberDuck app to be able to save a listing of a remote directory to a text file. If I am viewing a directory now and select the Print option, it creates a PDF, but it is like a PDF containing multiple screenshots.

1) In basic form, it would just include the file or folder name of the currently directory.
2) A nice addition would be for the output to include data from any additional columns I am viewing such as a date column, etc.
3) Another nice addition would be the ability to tell it to list not only the current files and folders, but include sub-folders and files in the output. This might be dangerous though if there were a lot of underlying items. 

Note: I believe that I can achieve saving a directory list by installing and using the CLI for CyberDuck, but it would be really nice to do this in the app.

**Comment by dkocher** [comment]
You can choose _Edit → Copy_ to copy the selected filenames to the clipboard.

**Reply by bowdendata** [reply]
Thanks! I don't know why I didn't try that, but it works well. One question though about dates. In my particular case, I am connecting to Dropbox and I have the Size and Modified (date) columns showing. In the output from the clipboard, the size values are in Bytes, so no problem there. The Modified column output is a number that appears to be UTC milliseconds? Is that correct?
